cdk/src/open-data-platform/lambda/lambda-layer-stack.ts

```diff
@@ -0,0 +1,63 @@
+import {Duration} from 'aws-cdk-lib';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as lambda from 'aws-cdk-lib/aws-lambda-nodejs';
+import * as rds from 'aws-cdk-lib/aws-rds';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import * as secretsmanager from 'aws-cdk-lib/aws-secretsmanager';
+import {Construct} from 'constructs';
+import * as path from 'path';
+
+interface SchemaProps {
+  cluster: rds.ServerlessCluster, vpc: ec2.IVpc, db: string,
+  credentialsSecret: secretsmanager.ISecret,
+}
+
+export class LambdaLayerStack extends Construct {
```

Comment:
Could we name this something more descriptive about its purpose? Like `DataImportStack`.